policykit/policyengine/views.py

```diff
@@ -130,7 +130,7 @@ def settings_page(request):
     user = get_user(request)
     community = user.community
 
-    if METAGOV_ENABLED and user.is_community_admin:
+    if user.has_perm("metagov.can_edit_metagov_config"):
```

Comment:
@amyxzhang I updated this PR as you suggested to create a custom Role (`"Metagov Admin"`) with a single permission (`"can_edit_metagov_config"`). right now users with `is_community_admin` get automatically assigned to this role. 

<img width="1295" alt="Screen Shot 2021-04-12 at 16 02 56" src="https://user-images.githubusercontent.com/5333982/114454766-f4df0680-9ba8-11eb-93a3-872bb4b0c15d.png">

Reply:
Looks great!